parsers/handleWordpressComponents.js

```diff
@@ -74,6 +74,56 @@ export function fixAccordions() {
   };
 }
 
+const findCaptionText = (figureCaption, key) => {
```

Comment:
`key` is always `'text'` - I think we can hard-code it. 
And let's call `figureCaption` `figureCaptionNodes` - so it's obvious it's an array of nodes.

Reply:
brill, will put that in